validation/ref/numeric/HHChannels.jl

```diff
@@ -0,0 +1,143 @@
+module HHChannels
+
+export Stim, run_hh
+
+using Sundials
+using SIUnits.ShortUnits
+
+immutable HHParam
+    c_m       # membrane spacific capacitance
+    gnabar    # Na channel cross-membrane conductivity
+    gkbar     # K channel cross-membrane conductivity
+    gl        # Leak conductivity
+    ena       # Na channel reversal potential
+    ek        # K channel reversal potential
+    el        # Leak reversal potential
+    q10       # temperature dependent rate coefficient
+              # (= 3^((T-T₀)/10K) with T₀ = 6.3 °C)
+
+    # constructor with default values, corresponding
+    # to a resting potential of -65 mV and temperature 6.3 °C
+    HHParam(;
+        #c_m    = 0.01nF*m^-2,
+        c_m    = 0.01F*m^-2,
+        gnabar = .12S*cm^-2,
+        ena    = 115.0mV + -65.0mV,
+        gkbar  = .036S*cm^-2,
+        ek     = -12.0mV + -65.0mV,
+        gl     = .0003S*cm^-2,
+        el     = -54.3mV,
+        q10    = 1
+    ) = new(c_m, gnabar, gkbar, gl, ena, ek, el, q10)
+
+end
+
+immutable Stim
+    t0        # start time of stimulus
+    t1        # stop time of stimulus
+    j         # stimulus current density
```

Comment:
Used `j` to emphasise that it is a current density rather than a current. Lower case `i` for current density is a safe convention? Happy to change.